examples/user_interaction/client/src/index.js

```diff
@@ -18,6 +18,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import { exportRootSpanAfterLoadEvent } from '@opencensus/web-all';
+import { startInteractionTracker } from '@opencensus/web-instrumentation-zone';
+
+// Necessary import for @opencensus/web-instrumentation-zone
+import { Zone, ZoneType, Task } from 'zone.js';
```

Comment:
I'm a bit surprised this line is needed here. What happens if you remove it?

Reply:
I already did that but compilation fails.

Reply:
OK, we can leave it in there for now. Longer term we will want to think about how to bring in Zone.js cleanly, but for now if this is needed we can keep it.